app/components/search/ResourceEntry.jsx

```diff
@@ -19,7 +19,18 @@ class ResourceEntry extends Component {
       <Link to={`/organizations/${hit.resource_id}`}>
         <li className="results-table-entry resource-entry">
           <div className="entry-details">
-            <h4 className="entry-headline">{`${hitNumber}. ${hit.name}`}</h4>
+            <div className="entry-header">
+              <h4 className="entry-headline">{`${hitNumber}. ${hit.name}`}</h4>
+              {hit.is_mohcd_funded
+                ? (
```

Comment:
Nit: If the "else" case just returns a `null`, then we can just use `&&`. E.g.

```jsx
{hit.is_mohcd_funded && <div>...</div>}
```

Reply:
oh ya, good call. I'll merge this one and update in the consolidated component!